src/components/Layout/styles/Chip.module.css

```diff
@@ -60,6 +60,10 @@
     flex: 1;
 }
 
+.chip:hover {
+    background: #9fdbd5;
```

Comment:
Is there no `ui` color available for this one?

Reply:
Nope. Needed something more subtle for a hover state than the difference between `200` and `300`. Could possibly introduce `250`, but doesn't seem needed just yet.